modules/installation-nutanix-ccm.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+
+:_content-type: PROCEDURE
+[id="nutanix-ccm-config_{context}"]
+= Adding the ConfigMaps and Secret needed for Nutanix CCM
+
+Installations on Nutanix require additional ConfigMaps and a Secret to integrate with the Nutanix Cloud Controller Manager (CCM).
```

Comment:
```suggestion
Installations on Nutanix require additional `ConfigMaps` and a `Secret` to integrate with the Nutanix Cloud Controller Manager (CCM).
```
See OpenShift docs guideline for [Object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references)

Reply:
+1, and if you choose to go with the backtick version for object references, make sure to also include the word "object" or "resource" or similar following the object names.